cluster/manifests/skipper/deployment.yaml

```diff
@@ -0,0 +1,92 @@
+{{ with $root := . }}
+{{ with $region := $root.Region }}
+{{ with $cfg := $root.ConfigItems }}
+{{ range $suffix := split $cfg.zone_suffixes "," }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: skipper-ingress-{{ $suffix }}
+  namespace: kube-system
+  labels:
+    application: skipper-ingress
+    version: v0.10.122
+    component: ingress
+spec:
+  selector:
+    matchLabels:
+      application: skipper-ingress
+      zone: {{ $region }}{{ $suffix }}
+  template:
+    metadata:
+      name: skipper-ingress-{{ $suffix }}
```

Comment:
Does that do anything? We usually don't have this field here.

Reply:
yes with a,b,c it won't overwrite the deplyoment

Reply:
changed